bark/benchmark/benchmark_runner.py

```diff
@@ -203,6 +203,7 @@ def _run_benchmark_config(self, benchmark_config, viewer=None, maintain_history=
             if not terminal:
                 if viewer:
                     viewer.drawWorld(world, scenario._eval_agent_ids, scenario_idx=benchmark_config.scenario_idx)
+                    viewer.saveFig("/tmp/viewer_{}.png".format(step))
```

Comment:
I would suggest to use a file-group and save the images there.
later these still can be accessed in the bazel-bin dir.